tarefundaccountdetails = TaRefundAccountDetails(user_id=114969201,merchant_ref="729611a1-0-TM120612204884-114969201",batch_ref="pay_OLRG2lnpae0JiR",note="note",tm_booking_id="123",refund_mode="UPI",phone="123",upi_id="kk@okaxis",pnr_number="2116275642")
>>> tarefundaccountdetails.save()
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/usr/src/Trainman/TripAssurance/models.py", line 526, in save
    super(TaRefundAccountDetails, self).save(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/db/models/base.py", line 807, in save
    self.save_base(using=using, force_insert=force_insert,
  File "/usr/local/lib/python3.9/site-packages/django/db/models/base.py", line 846, in save_base
    post_save.send(
  File "/usr/local/lib/python3.9/site-packages/django/dispatch/dispatcher.py", line 191, in send
    return [
  File "/usr/local/lib/python3.9/site-packages/django/dispatch/dispatcher.py", line 192, in <listcomp>
    (receiver, receiver(signal=self, sender=sender, **named))
  File "/usr/src/Trainman/TripAssurance/models.py", line 556, in send_booking_refund_details_whatsapp
    from irctc.booking_api import confirm_bank_details_whatsapp, confirm_upi_details_whatsapp
  File "/usr/src/Trainman/irctc/booking_api.py", line 37, in <module>
    from irctc.payment_api import Payment, GatewayRefund, GatewayRefundSerializer
  File "/usr/src/Trainman/irctc/payment_api.py", line 39, in <module>
    from irctc.utils import get_pg_base_and_tax, same_pg_charges_version
  File "/usr/src/Trainman/irctc/utils.py", line 10, in <module>
    from TmCash.tm_cash import issue_tm_cash_for_irctc_link
  File "/usr/src/Trainman/TmCash/tm_cash.py", line 15, in <module>
    from User.user_auth import User
  File "/usr/src/Trainman/User/user_auth.py", line 33, in <module>
    from Referral.referral import Referral
  File "/usr/src/Trainman/Referral/referral.py", line 19, in <module>
    from TmCash.tm_cash import TrainmanCash
ImportError: cannot import name 'TrainmanCash' from partially initialized module 'TmCash.tm_cash' (most likely due to a circular import) (/usr/src/Trainman/TmCash/tm_cash.py)
>>> 
